src/teleop/gui/src/components/ArmControls.vue

```diff
@@ -70,8 +129,54 @@ export default {
   methods: {
     updateArmEnabled: function (enabled) {
       this.arm_enabled = enabled;
+      //ra_command, have all nulls in position. And all zeros in velocity array
+      if (this.arm_enabled === false) {
+        const zeroCommandData = {
+          name: [
+            "joint_a",
+            "joint_b",
+            "joint_c",
+            "joint_d",
+            "joint_e",
+            "joint_f"
+          ],
+          position: [null, null, null, null, null, null],
+          velocity: [0, 0, 0, 0, 0, 0],
+          effort: [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
+        };
+        var zeroCommandMsg = new ROSLIB.Message(zeroCommandData);
+        this.jointstate_pub.publish(zeroCommandMsg);
```

Comment:
Wait does this go directly to esw? For one it seems unnecessary to add an extra publisher to this component but we also need to be careful to guarantee that this is the last message sent to esw

Reply:
Ok so after thinking through it a bit more, it would probably make more sense to cut the commands off at the jetson_teleop level instead. So we would have the arm enabled checkbox tied to a bool that would then be listened to by jetson_teleop. This way we can access this bool for the purpose of sending/not sending motion planned arm messages as well. I think that this change, however, is beyond the scope of this PR, so I removed the zeroing from the GUI for now and will make the change with the bool in jetson teleop later on.